setup.cfg

```
@@ -1,5 +1,5 @@
 [flake8]
 max_line_length = 88
 extend-ignore = E203, E231, E501, F541, D
-exclude = build,dist,*.egg-info,.venv,docs/_gallery
+exclude = build,dist,*.egg-info,.venv,docs/_gallery,docs_build
```

I guess there was supposed to be a slash in there?

```suggestion
exclude = build,dist,*.egg-info,.venv,docs/_gallery,docs/_build
```

---

Nope, it's `docs_build`. `docs/build` doesn't work because we don't want to build the docs inside the source files and `build` is taken by the actual build of pygfx.